src/ubloxGPS.cpp

```diff
@@ -1010,14 +1110,26 @@ bool ubloxGPS::getEsfStatus(ubx_esf_status_t &esf)
     return esf_status.valid;
 }
 
-bool ubloxGPS::setReset(void)
-{
-    // Hard coded currently to: Cold start, hardware reset (watchdog) immediately
+bool ubloxGPS::getSatelliteInfo(ubx_nav_sat_t &sats) {
     LOCK();
-    uint8_t sentences[8] = { (uint8_t)UBX_CLASS_CFG, (uint8_t)UBX_CFG_RST, 0x04, 0x00, 0xff, 0xff, 0x00, 0x00 };
-    return requestSendUBX(sentences, 8);
+    memcpy(&sats, &nav_sat, sizeof(ubx_nav_sat_t));
+    return true;
+}
+
+bool ubloxGPS::getOrbitDB(ubx_nav_orb_t &orb) {
+    LOCK();
+    memcpy(&orb, &nav_orb, sizeof(ubx_nav_orb_t));
+    return true;
 }
 
+// bool ubloxGPS::setReset(void)
```

Comment:
Is this a placeholder method?

Reply:
Ah, this is a leftover comment — I replaced this method with [the one at 1480](https://github.com/particle-iot/gps-ublox/pull/13/files/18c16763ba691ac72ccc99365a00372563c9d5e9#diff-4b85af269490da03eb14522cd1212ddb0c0b073ab3af95285f68f81636cd560fR1480)

This can be removed.